CSharp/DevVmPowershell/Helpers/Implementations/InstanceSettingsHelper.cs

```diff
@@ -110,6 +110,10 @@ public string GetInstanceSettingValue(string name, string section)
 					InstanceSettingQueryResultSet instanceSettingQueryResultSet = instanceSettingManager.QueryAsync(query).Result;
 					if (instanceSettingQueryResultSet.Success)
 					{
+						if (instanceSettingQueryResultSet.TotalCount == 0)
+						{
+							return null;
```

Comment:
Depending on how this is used, should we leave it at returning null, or should we throw an error?